src/utils/url-utils.ts

```diff
@@ -52,10 +36,16 @@ export function getSorteringsRekkefolgeFromUrl() {
     return queryString.parse(window.location.search).sorteringsrekkefolge || IKKE_SATT;
 }
 
-export function getPersonUrl(fnr: string, pathParam: string, enhet: string): string {
-    const enhetParam = enhet ? '?enhet=' + enhet : '';
-    const params = pathParam + enhetParam;
-    return `/veilarbpersonflatefs/${fnr}${params}`;
+export function getVeilarbpersonflateUrl(pathParam: string | null, enhet: string): string {
+    const basePath =
+        (process.env.REACT_APP_DEPLOYMENT_ENV as DeploymentEnvironment) === 'production'
+            ? 'https://veilarbpersonflate.intern.nav.no'
```

Comment:
nå ligger disse strengene i to forskjellige filer, vet ikke om det ligger flere steder i appen eller kommer til å trenges i endringer i fremtiden, men kan vurderes å lage konstanter 🤷‍♀️  men ikke no must